cmd/complytime/cli/info.go

```diff
@@ -0,0 +1,644 @@
+	"github.com/complytime/complytime/internal/complytime"
+	"github.com/complytime/complytime/internal/terminal"
+)
+
+const (
+	colWidthControlID      = 12
+	colWidthControlTitle   = 40
+	colWidthImplStatus     = 21
+	colWidthPluginsUsed    = 24
+	colWidthRulesInControl = 55
+	colWidthPluginUsed     = 31
+)
+
+var (
+	tableHeaderStyle = table.DefaultStyles().Header.
+				BorderStyle(lipgloss.NormalBorder()).
+				BorderForeground(lipgloss.Color("240")). // Light grey border
+				BorderBottom(true).
+				Bold(false)
+
+	tableCellStyle = table.DefaultStyles().Cell.
+			Foreground(lipgloss.Color("250")). // Very light grey text
+			Bold(true)
+
+	// Style for key-value pair keys
+	keyStyle = lipgloss.NewStyle().
+			PaddingRight(1)
+
+	// Style for key-value pairs values
+	valueStyle = lipgloss.NewStyle().
+			Foreground(lipgloss.Color("250")).
+			Bold(true)
+
+	// Container style for key-value information blocks
+	infoContainerStyle = lipgloss.NewStyle().
+				Border(lipgloss.NormalBorder()).
+				BorderForeground(lipgloss.Color("240")). // Light grey border for containers
+				Padding(0, 1).
+				Width(90) // Fixed width for consistent formatting
+)
+
+// Rule represents details about a rule for easy
+// mapping of rules to plugins.
+type Rule struct {
+	ID          string
+	Plugin      string
+	Description string
+	Parameters  []string
+}
+
+// Control repsents details about a control across
+// component sources.
+type Control struct {
+	ID                  string
+	Title               string
+	Description         string
+	ImplemenationStatus string
+	Rules               []Rule
+}
+
+type infoOptions struct {
+	*option.Common
+	complyTimeOpts *option.ComplyTime
+	controlID      string // show info for a specific control ID
+	ruleID         string // show info for a specific rule ID
+	limit          int    // limit number for table rows shown in terminal
+	plain          bool   // print plain table only
+}
+
+func infoCmd(common *option.Common) *cobra.Command {
+	infoOpts := &infoOptions{
+		Common:         common,
+		complyTimeOpts: &option.ComplyTime{},
+	}
+	cmd := &cobra.Command{
+		Use:     "info <framework-id> [flags]",
+		Short:   "Import OSCAL data from API",
+		Example: " complytime info anssi_bp28_minimal\n complytime info anssi_bp28_minimal --control r31\n complytime info anssi_bp28_minimal --rule enable_authselect",
+		Args:    cobra.ExactArgs(1),
+		PreRun: func(cmd *cobra.Command, args []string) {
+			if len(args) == 1 {
+				infoOpts.complyTimeOpts.FrameworkID = filepath.Clean(args[0])
+			}
+		},
+		RunE: func(_ *cobra.Command, _ []string) error { return runInfo(infoOpts) },
+	}
+	cmd.Flags().StringVarP(&infoOpts.controlID, "control", "c", "", "show info for a specific control ID")
+	cmd.Flags().StringVarP(&infoOpts.ruleID, "rule", "r", "", "show info for a specific rule ID")
+	cmd.Flags().IntVarP(&infoOpts.limit, "limit", "l", 0, fmt.Sprintf("limit the number of table rows"))
```

Comment:
```suggestion
	cmd.Flags().IntVarP(&infoOpts.limit, "limit", "l", 0, "limit the number of table rows")
```
Also flagged in lint test.